community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -236,67 +306,14 @@ def _seconds_since_timestamp(timestamp):
     return datetime.now().timestamp() - creation_dt.timestamp()
 
 
-def do_node_update(status, nodes):
-    """update node/instance based on node status"""
-    if status == NodeStatus.unchanged:
-        return
+def do_node_update(action, nodes):
+    log.info(action)
```

Comment:
It probably should be a single invocation of `log.info` instead of three.

Reply:
Had that for debugging didn't mean to add it to this commit, in fact we can remove `do_node_update` entirely and have it in-line with `sync_slurm`